packages/pouchdb-adapter-asyncstorage/src/bulk_docs.js

```diff
@@ -53,35 +54,59 @@ export default function (db, req, opts, callback) {
         })
       }
 
-      let binData
+      let resolveBinaryData;
```

Comment:
Use the `npm` and `node` versions from `travis` build since it's correctly installed there https://travis-ci.org/stockulus/pouchdb-react-native/jobs/178250553#L266-L271

```txt
$ node --version
v6.9.1
$ npm --version
3.10.8
```